designer/deployment-manager-api/src/main/scala/pl/touk/nussknacker/engine/testing/DeploymentManagerStub.scala

```diff
@@ -28,9 +28,9 @@ class DeploymentManagerStub extends BaseDeploymentManager with StubbingCommands
       lastStateAction: Option[ProcessAction]
   ): Future[ProcessState] = {
     val lastStateActionStatus = lastStateAction match {
-      case Some(action) if action.actionType.equals(ProcessActionType.Deploy) =>
+      case Some(action) if action.actionName.equals(ScenarioActionName.Deploy) =>
```

Comment:
I know that it isn't part of your change, but please change all ocrurrences of `equals` into `==`

Reply:
Fixed (replaced all `actionName.equals` )